.github/workflows/pull_request.yml

```diff
@@ -19,15 +10,36 @@ jobs:
         variant: [Debug, Release]
     steps:
 
-    - uses: actions/checkout@master
+    - name: Check if relevant files have changed
+      uses: actions/github-script@0.9.0
+      id: service-changed
+      with:
+        result-encoding: string
+        script: |
+          const result = await github.pulls.listFiles({
+            owner: context.payload.repository.owner.login,
+            repo: context.payload.repository.name,
+            pull_number: context.payload.number,
+            per_page: 100
+          })
+          const serviceChanged = result.data.filter(f => f.filename.startsWith("app/") || f.filename.endsWith("gradle") || f.filename.startsWith(".github") || (f.filename.startsWith("gradle") && f.filename.endsWith("properties"))).length > 0
```

Comment:
Don't we also need to run the checks when making changes to `gradlew`? Maybe the `&&` should just be an `||`.

Reply:
Yeah that's right. Fixed it.